### PR TITLE
update valkey in serverLog messeges in server.c file

### PR DESCRIPTION
**Pull request by Shivshankar-Reddy**
Updated keyword "Redis" to "Valkey" in log messeges in server.c file

### CODE REVIEW

**Review by hwware — APPROVED** [review]

**Comment by Shivshankar-Reddy** [comment]
@zuiderkwast Can you review this PR?

**Comment by zuiderkwast** [comment]
Partial implementation of #207.

Log messages are breaking changes. Let's remember for release notes.

File: src/server.c [hunk 1]
```diff
@@ -6811,10 +6811,10 @@ static int serverSupervisedUpstart(void) {
 static int serverSupervisedSystemd(void) {
 #ifndef HAVE_LIBSYSTEMD
     serverLog(LL_WARNING,
-            "systemd supervision requested or auto-detected, but Redis is compiled without libsystemd support!");
+            "systemd supervision requested or auto-detected, but Valkey is compiled without libsystemd support!");
     return 0;
 #else
-    if (serverCommunicateSystemd("STATUS=Redis is loading...\n") <= 0)
+    if (serverCommunicateSystemd("STATUS=Valkey is loading...\n") <= 0)
```
Comment (zuiderkwast) [inline src/server.c:6817]: This is a special message. Not a normal log entry. I'm marking it for special attention.

**Review by zuiderkwast — APPROVED** [review]
OK, LGTM

**Comment by zuiderkwast** [comment]
Oops, maybe we shouldn't merge this after the last discussions in #207.

No problem. We can just update these again.

**Pull request merged**